### PR TITLE
Accelerate makefile build speed

### PR DESCRIPTION
**Pull request by hunterlxt**
Signed-off-by: Xintao <hunterlxt@live.com>

### What is changed and how it works?

- Use the same parameters for `mem-profiling` test to avoid recompilation and speed up

### Release note 
- No

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -234,11 +234,13 @@ run-test:
 	export RUST_BACKTRACE=1 && \
 	cargo test --workspace \
 		--exclude fuzzer-honggfuzz --exclude fuzzer-afl --exclude fuzzer-libfuzzer \
-		--features "${ENABLE_FEATURES}" ${EXTRA_CARGO_ARGS} -- --nocapture && \
+		--features "${ENABLE_FEATURES} mem-profiling" ${EXTRA_CARGO_ARGS} -- --nocapture && \
 	if [[ "`uname`" == "Linux" ]]; then \
 		export MALLOC_CONF=prof:true,prof_active:false && \
 		cargo test --features "${ENABLE_FEATURES} mem-profiling" ${EXTRA_CARGO_ARGS} -p tikv_alloc -- --nocapture --ignored && \
-		cargo test --features "${ENABLE_FEATURES} mem-profiling" ${EXTRA_CARGO_ARGS} -p tikv --lib -- -- nocapture --ignored; \
+		cargo test --workspace \
```
Comment (BusyJay) [inline Makefile:241]: I'm afraid tests will be run twice.

Comment (hunterlxt) [reply Makefile:241]: which tests you mean? Can you point it out?

Comment (hunterlxt) [reply Makefile:241]: I just ran those tests using `make run-test` and didn't see repeated test result.

Comment (BusyJay) [reply Makefile:241]: It seems I misunderstood how `--ignored` worked.

**Comment by yiwu-arbug** [comment]
Can you filling the PR summary to describe what does the change do?

**Comment by hunterlxt** [comment]
Now merge operation needs sig-leader to trigger

**Comment by hunterlxt** [comment]
@ti-srebot Can you merge it again?

**Pull request merged**